src/components/UI/CardList.jsx

```diff
@@ -1,17 +0,0 @@
-import "./CardList.css";
-import Card from "./Card/Card";
-
-const CardList = (props) =>
-  props.cardsText.map((cardText) => (
-    <Card
-      key={cardText.id}
-      cardText={cardText}
-      className="card"
-      isViewMode={props.isViewMode}
-      onUpdatedCardText={(cardText) => {
-        props.onUpdatedCardText(cardText);
-      }}
-    ></Card>
-  ));
-
-export default CardList;
```

Comment:
You could actually set it as renaming action via git using specific keys while adding to index/commiting. But nevermind.